### PR TITLE
Added min-height to Popover.Pane

### PR DESCRIPTION
**Pull request by dleroux**
### WHY are these changes introduced?

To accommodate the new `TagAutocomplete` in web, a `height` prop was added to `Combobox`. This height prop gets passed down to the `maxHeight` and `height` of the `Popover.Pane`. However because the popover adds inline styles, a `minHeight` is also needed to prevent the popover from resizing based on viewport space. Without it, the `TagAutocomplete` overlay moved between the top and bottom of the Textfield based on its content which wasn't a great UX.

### WHAT is this pull request doing?

Adds `minHeight` to the `Popover.Pane` based on the height prop. The same approach used for the `maxHeight`.

### How to 🎩

🖥 [Local development instructions](https://github.com/Shopify/polaris/blob/main/README.md#local-development)
🗒 [General tophatting guidelines](https://github.com/Shopify/polaris/blob/main/documentation/Tophatting.md)
📄 [Changelog guidelines](https://github.com/Shopify/polaris/blob/main/documentation/Versioning%20and%20changelog.md)

I added a height tot the Single select autocomplete Combobox example here:
spin: http://240.16.151.141:6006/?path=/story/all-components-combobox--single-select-autocomplete

<details>
<summary>Copy-paste this code in <code>playground/Playground.tsx</code>:</summary>

```jsx
import React from 'react';
import {Page} from '../src';

export function Playground() {
  return (
    <Page title="Playground">
      {/* Add the code you want to test in here */}
    </Page>
  );
}
```

</details>

### 🎩 checklist

- [ ] Tested on [mobile](https://github.com/Shopify/polaris/blob/main/documentation/Tophatting.md#cross-browser-testing)
- [ ] Tested on [multiple browsers](https://help.shopify.com/en/manual/shopify-admin/supported-browsers)
- [ ] Tested for [accessibility](https://github.com/Shopify/polaris/blob/main/documentation/Accessibility%20testing.md)
- [ ] Updated the component's `README.md` with documentation changes
- [ ] [Tophatted documentation](https://github.com/Shopify/polaris/blob/main/documentation/Tophatting%20documentation.md) changes in the style guide
- [ ] For visual design changes, ping @ sarahill to update the Polaris UI kit

### CODE REVIEW

File: documentation/Releasing.md [hunk 1]
```diff
@@ -4,35 +4,35 @@ The process for releasing the monorepo's contents is in active development. This
 
 ### [polaris-for-vscode](/polaris-for-vscode)
```
Comment (dleroux) [inline documentation/Releasing.md]: @romellogoodman, I'm not sure why your change here: https://github.com/Shopify/polaris/pull/5676 didn't fail the linter.

Comment (romellogoodman) [reply documentation/Releasing.md]: Just pushed an update to main with these changes. Thanks for the catch

**Review by kyledurand — COMMENTED** [review]
Should minHeight be a new prop?

**Comment by dleroux** [comment]
> Should minHeight be a new prop?

I don't believe so, the same way maxHeight isn't. It's just to make sure the popover keep the height it was given.

**Comment by dleroux** [comment]
@romellogoodman @kyledurand can I get a ✅

**Review by kyledurand — APPROVED** [review]

**Pull request merged**